From d67c1e92a27e72c9aa4795f2fef1e83274c01083 Mon Sep 17 00:00:00 2001
From: Nico Madysa <nico.madysa@tu-dresden.de>
Date: Thu, 21 Feb 2019 23:19:23 +0100
Subject: [PATCH 1/2] Allow deprecated item std::ascii::AsciiExt.

Not allowing it breaks compatibility with nightly Rust 1.33.0. Removing
the statement breaks compatibility with stable Rust 1.3.0.

(cherry picked from commit 29b30b3bef2f3af06997e8d96b05d63509a49980)
---
 src/lib.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/lib.rs b/src/lib.rs
index 43a709c..481f2aa 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -21,6 +21,7 @@
 #[cfg(feature = "heap_size")]
 extern crate heapsize;
 
+#[allow(unused, deprecated)]
 use std::ascii::AsciiExt;
 #[cfg(__unicase__iter_cmp)]
 use std::cmp::Ordering;
-- 
2.24.0

